scripts/ci/testing/ci_run_airflow_testing.sh

```diff
@@ -31,14 +31,22 @@ fi
 function run_airflow_testing_in_docker() {
     set +u
     set +e
-    # shellcheck disable=SC2016
-    docker-compose --log-level INFO \
-      -f "${SCRIPTS_CI_DIR}/docker-compose/base.yml" \
-      -f "${SCRIPTS_CI_DIR}/docker-compose/backend-${BACKEND}.yml" \
-      "${INTEGRATIONS[@]}" \
-      "${DOCKER_COMPOSE_LOCAL[@]}" \
-         run airflow "${@}"
-    EXIT_CODE=$?
+    for TRY_NUM in {1..3}
+    do
+        docker-compose --log-level INFO \
+          -f "${SCRIPTS_CI_DIR}/docker-compose/base.yml" \
+          -f "${SCRIPTS_CI_DIR}/docker-compose/backend-${BACKEND}.yml" \
+          "${INTEGRATIONS[@]}" \
+          "${DOCKER_COMPOSE_LOCAL[@]}" \
+             run airflow "${@}"
+        EXIT_CODE=$?
+        if [[ ${EXIT_CODE} == 254 ]]; then
+            echo "Failed on starting integration on ${TRY_NUM} try."
```

Comment:
Should we wait some time before retrying?

Reply:
@feluelle Yeah. It won't change too much but sleeping a few seconds will help with flushing any buffers etc. 
Also I added one more step - I run 'docker-compose down --remove-orphans -v' which will shut down the network and anything else that remained from the previous docker-compose run (but the images remain). This way if an integration started already - it will be killed.

I think one part of the problem is an occasional race that happens between establishing the network by docker-compose and starting the integrations - when the images are being downloaded. So the second (and third) run of docker-compose will start with all images already downloaded and completely new network to set-up.

I think that should help.

Reply:
Sounds great! Let's give it a try (hopefully only 1/1) :)